woocommerce/wc-admin
# Update woo-dash naming to wc-admin

This PR updates the naming of `woo-dash`/`woo_dash` to their `wc-admin` counterparts. **Note:** This branch can cause your plugin to get deactivated since the name of the main PHP file is also changed. It's better to do that now before we open up the repo. It also changes the URL key and the text-domain so some branches might have to be rebased.

I didn't change the text-domain to `woocommerce` yet. I *think* we should wait until core merge to do that. That way language files can be generated for this plugin and not conflict with core files.

Fixes #138.

To Test:
* Apply this branch.
* Load `wp-admin`. The plugin will get deactivated because it will try to load `woo-dash.php`. Go to your plugins list and activate `WooCommerce Admin`. Now go to each screen (dashboard, analytics, revenue, etc) and verify you can navigate, the menus highlight, etc.
